.circleci/config.yml

```diff
@@ -207,4 +232,13 @@ workflows:
           filters:
             branches:
               only: main
```

Comment:
👋🏿 @davemcorwin - the trigger is running in main and staging? but the restage job only in main?
should job include staging? or the trigger only main?

Reply:
? There is a restage job for both main and staging

Reply:
yep, you're right 👍🏿